### PR TITLE
updated leftjoin

### PR DESCRIPTION
**Pull request by xialiu1988**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: 401code_challenges/codeChallenge/src/test/java/leftJoin/LeftJoinTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    List<String> list = Leftjointables(h1, h2);
+    String expected = "food:apple,null";
+         assertEquals(expected, list.get(0));
+
+}
+
+    // left table has drink, so right table value will be null for the drink
+
+    @Test
+    public void cancombinethevaluesbasedonlefttable() {
```
Comment (evanslaton) [inline 401code_challenges/codeChallenge/src/test/java/leftJoin/LeftJoinTest.java]: Remember to camelCase your variables/method names!
`canCombineTheValuesBasedOnLeftTable`
